packages/@ember/debug/lib/capture-render-tree.ts

```diff
@@ -20,7 +20,7 @@ import { expect } from '@glimmer/util';
   @since 3.14.0
 */
 export default function captureRenderTree(app: Owner): CapturedRenderNode[] {
-  let renderer = expect(app.lookup<Renderer>('renderer:-dom'), `BUG: owner is missing renderer`);
+  let renderer = expect(app.lookup('renderer:-dom') as Renderer, `BUG: owner is missing renderer`);
```

Comment:
**~:** could use a better assertion to check that it's not just present but is the right/expected thing.